src/main/java/frc/team3130/robot/commands/Chassis/DefaultDrive.java

```diff
@@ -15,13 +15,13 @@ public DefaultDrive() {
         this.subsystems = Set.of(Chassis.getInstance());
     }
 
-
     /**
      * The initial subroutine of a command.  Called once when the command is initially scheduled.
      */
     @Override
     public void initialize() {
-
+        System.out.println("Setting drive max ramp rate......");
```

Comment:
System out prints are expensive and potentially thread locking operations. All debug messages like this should be removed from the code as soon as they are not needed. To communicate messages back to the driver station it's safer to use DriverStation's "warn" method or the Smart/Shuffle boards.